src/cmds/openapi/uses.ts

```diff
@@ -0,0 +1,150 @@
+import type { CommandOptions } from '../../lib/baseCommand';
+
+import chalk from 'chalk';
+import ora from 'ora';
+import pluralize from 'pluralize';
+import { getBorderCharacters, table } from 'table';
+
+import analyzeOas from '../../lib/analyzeOas';
+import Command, { CommandCategories } from '../../lib/baseCommand';
+import { oraOptions } from '../../lib/logger';
+import prepareOas from '../../lib/prepareOas';
+
+export type Options = {
+  spec?: string;
+  workingDirectory?: string;
+};
+
+export default class OpenAPIUsesCommand extends Command {
+  constructor() {
+    super();
+
+    this.command = 'openapi:uses';
```

Comment:
I'm not sold on `:uses` for this command name and still wonder if `:inspect` or `:analyze` are better suited for what all it does.

Reply:
talked offline and we're going to go with `:inspect`